tests/integration/defs/conftest.py

```diff
@@ -1943,8 +1943,24 @@ def get_device_memory():
             "nvidia-smi" + suffix, "--query-gpu=memory.total",
             "--format=csv,noheader"
         ])
-        output = check_output(cmd, shell=True, cwd=temp_dirname)
-        memory = int(output.strip().split()[0])
+        try:
```

Comment:
I am guessing this is because of memory reading not working on 121, maybe we can add a comment to explain?